projects/uswds-components/src/lib/form-field/form-field.component.html

```diff
@@ -0,0 +1,15 @@
+<div #labelDiv [class.usa-form-group--error]="!!errorMessage" class="usa-form-group ">
+  <label *ngIf="!!label" [attr.for]="name" class="usa-label" [class.usa-input-error-label]="errorMessage">
+    {{ label || '' }}
+    <span *ngIf="required || requiredFlag">
+      (Required)
+    </span>
+    <ng-content select="[label-right]"></ng-content>
+  </label>
+  <small *ngIf="description" [attr.id]="this.name + '-description'" class="form-text text-muted"
```

Comment:
I see that we're using name property here instead of ID. Could we default name to some auto-incrementing property? Currently, name is undefined by default, so the id would be undefined + 'description'. If we default name to empty string, id would be '-description', which is OK, but problems could arise if there are multiple form-fields in the same page, which could cause duplicate ID. So whether we decide to rely on name or errorElId, let's make sure that value has something defined by default, and that it will also be unique

Reply:
Updated the attribute with id value